### DISCUSSION TITLE
[General] What is the Blocktime of Zksync ERA and it's testnet ? I can't figure it out.

### DISCUSSION THREAD

**Discussion by afa7789** [zkSync-Community-Hub/zksync-developers#113] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I'm trying to add the chain to Tally. And I kind of need to know the blcok time.

### What is the specific issue or error you're encountering?

... I can't figure the block time from explorer, and this info is not displayed anywhere.

### Can you share the error messages or logs you're receiving, if any?

I dont have

### Have you made any recent changes to the contract before encountering this issue?

N/a

### Are there any external libraries or contracts that your contract interacts with?

n/a

### Can you provide the relevant portions of your contract code where the issue is occurring?

n/a

### Have you tried to isolate the problem, and if so, what were the results?

n/a

### What steps have you already taken to try to resolve the issue?

n/a

**Comment by Romsters** [comment, marked as answer]
Hi @afa7789, when transactions are made on zkSync Era, they are processed right away and added to blocks, which are generated immediately. Once zkSync Era becomes fully decentralised, block time will take a couple of seconds, as the involved entities would need to achieve consensus.

**Comment by threadaqged** [comment]
To determine the block time of a blockchain, you can typically find this information in the blockchain's documentation, or by directly querying the blockchain's API. Additionally, community forums or developer channels related to the blockchain may provide insights into block time. If the information is not readily available, you can estimate the block time by analyzing the average time between consecutive blocks.

**Comment by adambold** [comment]
To determine the block time on a blockchain, you can calculate the average time between blocks by analyzing the timestamps of consecutive blocks. This can be done programmatically by fetching block data from a blockchain explorer or node API and computing the time differences between blocks.

**Comment by shelleyalexander** [comment]
To find the block time of a blockchain, you can usually check the documentation of the blockchain platform or search for it online. Alternatively, you can inquire with the blockchain community or developers familiar with the platform.

**Comment by seimabirthcan** [comment]
You can typically find the block time of a blockchain network by checking its documentation or by querying the network directly through a node or an API. If you're unable to find this information, you can try reaching out to the community or the developers of the blockchain network for assistance.

**Comment by mooralfeapur** [comment]
The user is trying to add the chain to Tally on the zkSync Mainnet and needs to know the block time. However, the user cannot find this information on the explorer or anywhere else. No error messages or logs are provided, and no recent changes have been made to the contract. No external libraries or contracts are mentioned, and no relevant portions of the contract code are provided. No steps have been taken to isolate the problem, and no repo link is provided. No additional details are provided.
